parcel/Example_P3_vs_activitybased.jl

```diff
@@ -0,0 +1,217 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+
+# definition of the ODE problem for parcel model
+include(joinpath(pkgdir(CM), "parcel", "Parcel.jl"))
+FT = Float32
+# get free parameters
+tps = TD.Parameters.ThermodynamicsParameters(FT)
+aps = CMP.AirProperties(FT)
+wps = CMP.WaterProperties(FT)
+ip = CMP.IceNucleationParameters(FT)
+
+# Constants
+ρₗ = wps.ρw
+R_v = TD.Parameters.R_v(tps)
+R_d = TD.Parameters.R_d(tps)
+
+# Initial conditions
+Nₐ = FT(2000)
+Nₗ = FT(2000)
+Nᵢ = FT(0)
+rₗ = FT(1.25e-6)
+p₀ = FT(20000)
+T₀_dep = FT(238)
+T₀_het = FT(239)
+T₀_hom = FT(236.5)
+qᵥ = FT(8.3e-4)
+qₗ = FT(Nₗ * 4 / 3 * π * rₗ^3 * ρₗ / 1.2)
+qᵢ = FT(0)
+x_sulph = FT(0)
+
+# Moisture dependent initial conditions
+q = TD.PhasePartition(qᵥ + qₗ + qᵢ, qₗ, qᵢ)
+ts_dep = TD.PhaseNonEquil_pTq(tps, p₀, T₀_dep, q)
+ts_het = TD.PhaseNonEquil_pTq(tps, p₀, T₀_het, q)
+ts_hom = TD.PhaseNonEquil_pTq(tps, p₀, T₀_hom, q)
+ρₐ_dep = TD.air_density(tps, ts_dep)
+ρₐ_het = TD.air_density(tps, ts_het)
+ρₐ_hom = TD.air_density(tps, ts_hom)
+Rₐ = TD.gas_constant_air(tps, q)
+eₛ_dep = TD.saturation_vapor_pressure(tps, T₀_dep, TD.Liquid())
+eₛ_het = TD.saturation_vapor_pressure(tps, T₀_het, TD.Liquid())
+eₛ_hom = TD.saturation_vapor_pressure(tps, T₀_hom, TD.Liquid())
+e = qᵥ * p₀ * R_v / Rₐ
```

Comment:
Could you replace this with the function defined in `ParcelCommon.jl`? https://github.com/CliMA/CloudMicrophysics.jl/blob/c53aef88084caac64350c96330382f687cbce786/parcel/ParcelCommon.jl#L7

The same for ksi and S_i defined below. This helps ensure we don't have any typos in the initial conditions. (or I guess if we do then it's the same typo for all...)